spec/datetimeformat.html

```diff
@@ -142,6 +146,31 @@ <h1>InitializeDateTimeFormat ( _dateTimeFormat_, _locales_, _options_ )</h1>
         1. Return _dateTimeFormat_.
       </emu-alg>
     </emu-clause>
+
+    <emu-clause id="sec-formattimezoneoffsetstring" aoid="FormatTimeZoneOffsetString">
+      <h1>FormatTimeZoneOffsetString ( _offsetNanoseconds_ )</h1>
+      <emu-alg>
+        1. Assert: _offsetNanoseconds_ is an integer.
```

Comment:
I'd recommend a structured header (and then there's no need for the assertion). We can also limit the type of _offsetNanoseconds_ to "an integer in the range of -86,400,000,000,000 to 86,400,000,000,000". (I forget whether the range is inclusive or exclusive at both ends.)

Reply:
This operation is defined in Temporal and will be removed from ECMA-402 and/or this PR once added to ECMA-262. I will propagate any upstream changes.